week1/homework/src/server.js

```diff
@@ -10,11 +10,50 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
-  });
+    response.setHeader('Content-Type', 'application/json');
+    switch (request.url) {
+      case '/state':
+        state = 10;
```

Comment:
You're reassigning `10` to `state` here. That means that this route does the same thing as the `/reset` route, i.e. resetting `state` to `10`.